### ISSUE TITLE
API should allow successful response of a blocking invoke to be just the result object 

### ISSUE THREAD

**Issue by rabbah** [openwhisk/openwhisk#132]
The current action activation API states that a successful blocking invoke responds with an activation record, which contains among various properties, the result of the activation. This is not convenient if the result of the activation needs to be consumed by a third party service. Hence, the API should provide an option to project the result from the activation on the server.

Note: The CLI provides a `--result` option which projects the result response from the activation result on the client side.

**Comment by vinodmut** [comment]
A projection on the activation `result` makes sense if we stop there. We should be careful about feature creep with requirements on settings response headers, mime types, etc. Such capabilities should belong in a separate service.

**Comment by fxulusoy** [comment]
Current activation result is on the following: 

GET .../activations/123
```
{
  "name": "echo",
  "subject": "guest",
  "activationId": "123",
  "publish": false,
  "annotations": [],
  "version": "0.0.1",
  "response": {
    "result": {
      "message": "Hello"
    },
    "success": true,
    "status": "success"
  },
  "end": 1459520294085,
  "logs": [],
  "start": 1459520294077,
  "namespace": "guest"
}
```

@rabbah , do you envision something like that? 

GET  .../activations/123?excludeMetadata=true
```
"response": {
    "result": {
      "message": "Hello"
    },
    "success": true,
    "status": "success"
  },
```

On the CLI:
```
wsk -v activation get 123 --excludeMetadata
```

**Comment by rabbah** [comment]
This is related to the `actions` API not the `activations` API endpoints. A request to invoke an action via `POST /actions/name?blocking=true` currently returns an activation record `W` as the one you showed above. Instead I am suggesting `POST /actions/name?blocking=true&result=true` to return `W.response.result` instead for a successful activation (status code 200).

**Comment by rabbah** [comment]
Fixed by #432.
